src/components/CVE-2021-44228/CVE-2021-44228.tsx

```diff
@@ -1,124 +1,243 @@
+     */
+    const killProcess = async () => {
+        if (!pid) {
+            alert("Nothing to kill.");
+            return;
+        }
+
+        try {
+            //Execute the killProcess command using the pid
+            await CommandHelper.runCommand("kill", [pid]);
+            //Set output and display message with pid if killed
+            setOutput((prevOutput) => prevOutput + `\nProcess ${pid} is killed and is dead.`);
+            //Set Pid after killing process
+            setPid("");
+            //Catch errors
+        } catch (error) {
+            //Set output and catch errors and display pid and display error message
+            setOutput((prevOutput) => prevOutput + `\nFailed to kill process with ${pid}: ${(error as Error).message}`);
+        }
+    };
+
     return (
-        <form onSubmit={form.onSubmit((values) => onSubmit(values))}>
-            <LoadingOverlay visible={loading} />
-            <Stack>
-                {UserGuide(title, description_userguide)}
-                <TextInput label={"Attacker IP Address"} required {...form.getInputProps("attackerip")} />
-                <NumberInput label={"Web Port"} required {...form.getInputProps("webport")} />
-                <NumberInput label={"Local Port"} required {...form.getInputProps("localport")} />
-                <Button type={"submit"}>Exploit</Button>
-                {SaveOutputToTextFile_v2(output, allowSave, hasSaved, handleSaveComplete)}
-                <ConsoleWrapper output={output} clearOutputCallback={clearOutput} />
-                {handle && (
-                    <Button leftIcon={<IconDoorExit />} onClick={killHandle} color={"red"}>
-                        Kill process
+        <RenderComponent
+            title={title}
+            description={description}
+            steps={steps}
+            tutorial={tutorial}
+            sourceLink={sourceLink}
+        >
+            {!loadingModal && (
+                <InstallationModal
+                    isOpen={opened}
+                    setOpened={setOpened}
+                    feature_description={description}
+                    dependencies={dependencies}
+                ></InstallationModal>
+            )}
+
+            <form onSubmit={form.onSubmit(onSubmit)}>
+                <Stack>
+                    {LoadingOverlayAndCancelButton(loading, pid)}
+                    <TextInput label={"Attacker IP Address"} required {...form.getInputProps("attackerIP")} />
+                    <NumberInput label={"Web Port"} required {...form.getInputProps("webPort")} />
+                    <NumberInput label={"Local Port"} required {...form.getInputProps("localPort")} />
+                    <Button type={"submit"}>Exploit</Button>
+                    <Button type={"button"} onClick={killProcess} disabled={!pid}>
+                        Kill Process
                     </Button>
```

Comment:
```suggestion
                    <Button type={"button"} onClick={killProcess} disabled={!pid}>Kill Process</Button>
```

Reply:
Tried fixing but tunning prettier after adding fix but it formats the killprocess button as it was before.